README.md

```diff
@@ -245,6 +245,17 @@ either you or they can follow these steps:
 
 This section covers the integration of GPT-4 Vision with Azure AI Search. Learn how to enhance your search capabilities with the power of image and text indexing, enabling advanced search functionalities over diverse document types. For a detailed guide on setup and usage, visit our [Enabling GPT-4 Turbo with Vision](docs/gpt4v.md) page.
 
+### Enabling Integrated Vectorization
+
+Azure AI search has introduced an [integrated vectorization feature in preview mode](https://techcommunity.microsoft.com/t5/ai-azure-ai-services-blog/announcing-the-public-preview-of-integrated-vectorization-in/ba-p/3960809#:~:text=Integrated%20vectorization%20is%20a%20new%20feature%20of%20Azure,pull-indexers%2C%20and%20vectorization%20of%20text%20queries%20through%20vectorizers). This feature is a cloud-based approach to data ingestion, which takes care of document format cracking, data extraction, chunking, vectorization, and indexing, all with Azure technologies.
+
+To enable integrated vectorization with this sample:
+
+1. If you've previously deployed, delete the existing search index.
+2. Run `azd env set USE_FEATURE_INT_VECTORIZATION true`
+3. Run `azd up` to update system and user roles
+4. You can view the resources such as the indexer and skillset in Azure Portal and monitor the status of the vectorization process.
```

Comment:
Please also add a section to data_ingestion.md about "Overview of the integrated vectorization process" that describes what gets created when you use this feature (skillset, vectorizer, etc). Include how someone would remove and add new data as that comes up constantly with the manual approach. And link to that from here. Thank you!